packages/core/src/bundle.ts

```diff
@@ -129,6 +129,23 @@ export async function bundleDocument(opts: {
     });
   }
 
+  if (config.resolveAfterTransformers) {
```

Comment:
I was thinking about it a bit. Let's not have this config.

Let's enable it if only there are any preprocessors enabled.

We don't have too many users using preprocessors I believe.
We should just add a little warning somewhere in the docs that using preprocessors has a performance penalty.